src/components/card/Card.jsx

```diff
@@ -3,12 +3,27 @@ import PropTypes from 'prop-types'
 import { Button, Card as UICard, Dropdown, Icon, Menu } from 'antd'
 import Modal from '../../commons/modal/Modal'
 import './style.css'
+import EditField from '../../commons/editField/EditField'
+
 
 class Card extends Component {
+  constructor(props) {
```

Comment:
The `this.state` is useless, so the constructor also. You can remove it entirely.